Dockerfile

```diff
@@ -1,50 +1,41 @@
-FROM python:3.5
-MAINTAINER Pit Kleyersburg <pitkley@googlemail.com>
-
-# Install dependencies
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        sudo \
-        tesseract-ocr tesseract-ocr-eng imagemagick ghostscript unpaper \
-    && rm -rf /var/lib/apt/lists/*
-
-# Install python dependencies
-RUN mkdir -p /usr/src/paperless
+FROM alpine:3.7
 WORKDIR /usr/src/paperless
```

Comment:
In light of reducing the image layers, please remove this `WORKDIR` instruction. This requires a change later on in the `RUN` instruction, I have added a comment on what I think is required.